### DISCUSSION TITLE
Content disapper after a second

### DISCUSSION THREAD

**Discussion by kstraszewski** [nuxt/nuxt#17756] [Questions]
Hi i've came across weird bug. After adding baseUrl ( `/b2c` in this case) the app start to act weird. Here is example:

https://qa2.lendi.pl/b2c/typography

App disappear after a sec. I've realized that entry-<hash>.js adds <layout_name_hash>.js and .css without baseUrl and i received 404. Also I can see that all of the chunks every layout prefetch at the beginning and i do not know why.

Any ideas?

**Comment by danielroe** [comment]
Could you open an issue with a reproduction? 🙏

**Reply by kstraszewski** [reply]
Here is a reproduction of the bug. It apears when add BaseUrl:

https://github.com/kstraszewski/nuxt3-playground

just do `npm run build` and `nuxi preview` Then go to http://localhost:3000/base/test and http://localhost:3000/base/testVuetify

You will see that app prefetch all CSS and js files and also some of them loads twice with different URL (one with baseURL and another one without) 

with baseUrl: 
<img width="714" alt="image" src="https://user-images.githubusercontent.com/10195995/161849122-d21bf41d-89c1-4c06-8b79-53cc367fa95d.png">

without:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/10195995/161848606-051de146-ba19-46f4-9c64-2a210bebc6db.png">

**Comment by kstraszewski** [comment]
Removing setup syntax from pages fixes this issue. But I do not know why :/ But prefetching all chunks and assets are still an issue for me. 

Edit: Nevermind, it wasn't that.

**Comment by kstraszewski** [comment]
Adding VThemeProvider component from Vuetify makes Error appear again. I think something is wrong with Nuxt 3 styles loader. 
This makes this error. Nuxt 3 adds additional prefetch of .mjs and .css for component when some weird error appear. It appears when using setup syntax inside pages and VThemeProvider, so I think there could be more of this.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/10195995/161628342-341f8afa-f59b-431c-aafe-09243a4f2a7b.png">
